### PR TITLE
Autodesk: Android Proposal

### PR DESCRIPTION
**Pull request by erikaharrison-adsk**
### Description of Proposal

This is [a proposal](https://github.com/autodesk-forks/USD-proposals/blob/adsk/feature/android/proposals/android/README.md) on supporting Android within OpenUSD. 

### Supporting Materials

N/A

### Contributing

- [X] I agree to and accept the [Supplemental Terms](https://graphics.pixar.com/usd/release/contributing_supplemental.html).

### CODE REVIEW

File: proposals/android/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Currently there is no Android demo app
+
+
+
+# Implementation 
+
+## Development environment
+- IDE: Android Studio Flamingo | 2022.2.1 Patch 2
+- Development Kit: NDK 25.2.9519653
+
```
Comment (meshula) [inline proposals/android/README.md:20]: Is there anything to say about support? Like, is this version a minimum expectation, or is it expected to be the only version that's supported?

File: proposals/android/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Changes/Development:
+- Compiled all core USD components on Android
```
Comment (meshula) [inline proposals/android/README.md:23]: This is complete, I'm guessing by the notes above, and the screenshot?

File: proposals/android/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- Add the Android specific Vulkan features to hgiVulkan
```
Comment (meshula) [inline proposals/android/README.md:24]: I imagine this, and the following, are areas of future development?

**Review by meshula — CHANGES_REQUESTED** [review]
I have a few questions ~ I was wondering if perhaps the PR could be updated to make those aspects more clear?

File: proposals/android/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+As we know, the importance of mobile devices graphics is increasing for this digital world. There are various types of mobile devices that require presentation, interaction, visualization, and other related graphics  is one of major mobile device markets.
+We propose a new rendering pipeline to extend the USD capabilities to support Android platform with Vulkan backend. The new solution helps benefit Android users from the USD ecosystem.
```
[the remark this answers is not in the document]
Comment (kaiquan-autodesk) [reply proposals/android/README.md:4]: This proposal is to introduce the ability to make USD work on Android.
The USD and Hydra/Hgi changes represent a complete Android workflow, which seems reasonable for one proposal.
 We’ll update it shortly with a section focused on the Hydra/Hgi side, which relates to our HgiVulkan proposal: https://github.com/PixarAnimationStudios/OpenUSD-proposals/pull/15.

File: proposals/android/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+# Technical challenges:
+
+- USD does not support Android including build system, rendering pipeline, lib plugin, file loader system and so on
```
[the remark this answers is not in the document]
Comment (kaiquan-autodesk) [reply proposals/android/README.md:8]: We already have a solution that we can load USD files on Android and render it to the screen with Vulkan backend. We’re working on tidying it up, and will send it out shortly.

File: proposals/android/README.md (hunk 1 above)
Comment (kaiquan-autodesk) [reply proposals/android/README.md:20]: This NDK 25.2.9519653 represents this initial Android implementation and what we've validated against. In time, we anticipate building on this work to support different versions - TBD. We welcome any additional solutions to get there sooner.

File: proposals/android/README.md (hunk 2 above)
Comment (kaiquan-autodesk) [reply proposals/android/README.md:23]: Yes, and we'll upload a related screenshot shortly.

File: proposals/android/README.md (hunk 3 above)
Comment (kaiquan-autodesk) [reply proposals/android/README.md:24]: This feature has been added, and we can now render USD files on Android. For future works, please see the section further down.

**Comment by erikaharrison-adsk** [comment]
Changes are reflected in the proposal. @meshula Please advise what other questions you have.

**Review by meshula — APPROVED** [review]

**Review by meshula — APPROVED** [review]

**Comment by meshula** [comment]
Hi Erika, the changes look good.  Unfortunately one of the updated commits didn't have signing, so merging is blocked. Would it be possible to rewrite the commits with signing, and then we can land it?

**Comment by erikaharrison-adsk** [comment]
Updated to verified commits.

**Pull request merged**